### DISCUSSION TITLE
How to open the file explorer

### DISCUSSION THREAD

**Discussion by ofzo** [helix-editor/helix#9479] [General] [answered]
Helix has or not file explorer like vscode

**Comment by TornaxO7** [comment, marked as answer]
Hi! Helix doesn't have a file explorer yet. It's planned to be added as a plugin (sauce: https://github.com/helix-editor/helix/pull/5768#issuecomment-1722508888), so you need to wait for the plugin system first.

**Reply by ofzo** [reply]
Got, do you know when the feature will release ?

**Reply by TornaxO7** [reply]
> Got, do you know when the feature will release ?

No, but you can keep track on it: https://github.com/helix-editor/helix/pull/8675

**Reply by TornaxO7** [reply]
@ofzo (please mark your question as answered if it helped you ;))

**Comment by Dangerise** [comment]
I think the best solution for now is to use helix in some tools like  [zellij](https://github.com/zellij-org/zellij) which can cut your terminal into some block, then you can run helix in a block and run a terminal file explore in another

I recommend you to use [yazi](https://github.com/sxyazi/yazi) as the file explore

**Reply by Stef16Robbe** [reply]
To add to this; yazi has added a way to configure this in their docs: https://yazi-rs.github.io/docs/tips/#helix-with-zellij

This opens a floating pane window with zellij, running yazi.

Maybe this can also be added in helix docs, in a FaQ section or something?

**Comment by tryoxiss** [comment]
It seems a file explorer was just merged in #11285 ! Posting this here since this is what always comes up when I check the status of the file browser (over the pr that added it, though thats new so it may get indexed eventually), so hopefully anyone else coming accross this can see the latest info.
